.pre-commit-config.yaml

```diff
@@ -2,11 +2,15 @@ ci:
   autoupdate_schedule: monthly
 exclude: '.*\.(css|js|svg)$'
 repos:
-  - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: 'v0.0.274'
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.0.275
```

Comment:
@frostming I think there is something wrong with ruff, because I have the same config for OpenLLM and ruff is correctly respect the single line imports when running within hatch and pre-commit

Reply:
Agree, it should have something to do with the parser. Ruff's parser is borrowed from RustPython, which may have different behaviors from CPython's.

When the parser fails, it quits silently

Reply:
hmm. Not to strongly on this, we can add isort back for now.